src/data/markdown/docs/02 javascript api/07 k6-experimental/02 redis.md

```diff
@@ -27,7 +27,4 @@ which operate in a synchronous manner,
 the Redis `Client` operates in an asynchronous manner.
 In practice, this means that using the Redis `Client`'s methods won't block test execution,
 and that the test will continue to run even if the Redis `Client` isn't ready to respond to the request.
-
-This new execution model does introduce a potential caveat: to depend on operations against Redis, all following operations should be made in the context of a [promise chain](https://javascript.info/promise-chaining).
-As the examples below demonstrate, whenever there is a dependency on a Redis operation result or return value, the operation should be wrapped in a promise itself.
-This way, a user can perform asynchronous interactions with Redis in a seemingly synchronous manner.
\ No newline at end of file
+The `async/await` syntax can be used to make the code look synchronous while still being asynchronous, and not blocking the test execution.
```

Comment:
I'm not exactly sure what this means, I'm afraid. Probably a result of my own ignorance.

But it sounds like it's really just about the smoother syntax, right? Maybe

> With `async/await` syntax, you can write asynchronous functions with less code and without blocking test execution.

What do you think? Otherwise, I have nothing to comment :-).

Reply:
Thanks for the feedback @MattDodsonEnglish, it's valuable to hear that someone who's not exposed to the syntax everyday doesn't necessarily understand how it's explained 😸 🙇🏻 

The description you gave is not really accurate I believe. I've been skimming the web to try and find a more accurate, yet more user-friendly one. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function)'s seems like a good candidate with " The async and await keywords enable asynchronous, promise-based behavior to be written in a cleaner style, avoiding the need to explicitly configure promise chains.". What do you think?

Reply:
I added it as a suggestion:

https://github.com/grafana/k6-docs/pull/1061#discussion_r1149104594

Reply:
Oh, I didn't know that! TIL 🙇🏻